### PR TITLE
Upstream OLM and API sync 12-14

### PR DESCRIPTION
**Pull request by dinhxuanvu**
Test #229 without OLMConfig PR

### CODE REVIEW

**Review by anik120 — APPROVED** [review]
/lgtm
trying 
/hold cancel 
since looks like maintainers don't have permission to edit PRs

**Comment by timflannagan** [comment]
/retitle Upstream OLM and API sync 12-14

**Pull request merged**
